api_tests/base/test_pagination.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+from nose.tools import *  # flake8: noqa
+
+from tests.base import ApiTestCase
+
+from api.base.pagination import MaxSizePagination
+
+class TestMaxPagination(ApiTestCase):
+    def test_no_query_param_alters_page_size(self):
+        assert_is_none(MaxSizePagination.page_size_query_param)
```

Comment:
Put an error message on this to say, "Adding variable page sizes to the paginator requires tests to ensure that you can't request more than the class's maximum number of values." So if they break it, they know what they need to do to fix it, and if it's removed in a PR, that the code reviewer will have a chance to see what it should have been replaced by.